client/src/App.tsx

```diff
@@ -123,6 +127,27 @@ function App() {
 
     useFetchInterceptor();
 
+    useEffect(() => {
+        analytics.init();
+    }, [analytics]);
+
+    useEffect(() => {
+        helpHub.init();
+    }, [helpHub]);
+
+    useEffect(() => {
+        if (authenticated) {
+            if (account) {
+                helpHub.boot(account);
+                helpHub.addRouter();
+            }
+        }
+
+        return () => {
+            helpHub.shutdown();
+        };
```

Comment:
Ovo mi  je čudno, zašto returnamo arrow funkciju koja radi `helpHub.shutdown()`. 
Nešto smrdi, probaj jel radi bez return-a, samo `helpHub.shutdown();` - tribalo bi